.eslintrc.json

```diff
@@ -121,7 +121,7 @@
   "overrides": [
     {
       "files": [
-        "**/*.spec.ts"
+        "**/*.unite-test.ts"
```

Comment:
Typo unite -> unit

spec.ts is naming convention used by nestjs, not sure about other frameworks.
unit-test.ts is also clear, so I don't mind the change.

Reply:
Thanks @avaitonis have spoke to @IgnatG and we are going to keep it consistent with other services and industry standard.

* `test.ts` : Unit test
* `api-test.ts`: API test
* `spec.ts`: E2E